src/content/es/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -13,137 +13,143 @@ book_path: /web/fundamentals/_book.yaml
+Warning: para simplificar este laboratorio de código y explicar los fundamentos de proporcionar una experiencia sin conexión, estamos usando JavaScript de vainilla. En una aplicación de producción, le recomendamos que utilice herramientas como [Workbox](/web/tools/workbox/) para construir tu service worker. Elimina muchos de los bordes afilados y las esquinas oscuras con las que puede encontrarse.
 
 ### Lo que aprenderás
 
 * Cómo crear y agregar un manifiesto de aplicación web.
-* Cómo proporcionar una experiencia fuera de línea simple
-* Cómo proporcionar una experiencia fuera de línea completa
+* Cómo proporcionar una experiencia sin conexión simple
+* Cómo proporcionar una experiencia sin conexión completa
 * Como hacer instalable tu aplicación
 
-Este código está enfocado en aplicaciones web progresivas. Los conceptos y los bloques de código no relevantes se pasan por alto y se proporcionan para que usted simplemente copie y pegue.
+Este laboratorio de código está enfocado en Progressive Web Apps. Los conceptos y los bloques de código no relevantes se pasan por alto y se proporcionan para que usted simplemente copie y pegue.
 
 ### Lo que necesitarás
 
-* Una versión reciente de los PWA de Chrome (74 o posterior) son solo aplicaciones web y funcionan en todos los navegadores, pero usaremos algunas funciones de Chrome DevTools para comprender mejor lo que está sucediendo en el nivel del navegador y usarlo para prueba la experiencia de instalación.
+* Una versión reciente de Chrome (74 o posterior) las PWAs son solo aplicaciones web y funcionan en todos los navegadores, pero usaremos algunas funciones de Chrome DevTools para comprender mejor lo que está sucediendo a el nivel de navegador y usarlo para probar la experiencia de instalación.
 * Conocimiento de HTML, CSS, JavaScript y [Chrome DevTools](https://developer.chrome.com/devtools) .
 
-## Preparándose
+## Preparación
 
 ### Consigue una clave para la API de Dark Sky
 
-Nuestros datos meteorológicos provienen de [Dark Sky API](https://darksky.net/dev) . Para poder usarlo, deberás solicitar una clave API. Es fácil de usar y gratis para proyectos no comerciales.
+Nuestros datos meteorológicos provienen de [Dark Sky API](https://darksky.net/dev). Para poder usarlo, deberás solicitar una API key. Es fácil de usar y gratis para proyectos no comerciales.
 
 [Register for API Key](https://darksky.net/dev/register)
 
-Note: Aún puedes completar este código de código sin una clave de API de Dark Sky. Si nuestro servidor no puede obtener datos reales de la API de Dark Sky, devolverá datos falsos en su lugar.
+Note: También puedes completar este laboratorio de código sin una API key de Dark Sky. Si nuestro servidor no puede obtener datos reales de la API de Dark Sky, devolverá datos falsos en su lugar.
```

Comment:
Note -> Nota

Reply:
They said: Please leave lines starting with "Caution:", "Note:", "Warning:", "Success:", "Key Point:", "Key Term:", and "Objective:" untranslated, they are special keyword that does some extra formatting.